Joyal.StaffManagement.Console/Program.cs

```diff
@@ -1,27 +1,68 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
+using DataLibrary;
+using System.Configuration;
+using DocumentFormat.OpenXml;
+using StaffDatabaseHelper;
 
-namespace staff_management_2
+namespace StaffManagementConsole
 {
     class Program
     {
+        public static int ID { get; set; }
         
-         public static int ID { get; set; }
-        public static List<Staff> StaffList = new();
-        public static bool IsAStaff(int id)
+        private static List<IStaffOperation> staffList = new();
+        static string SQLConnectionString = ConfigurationManager.AppSettings["SQLConnectionString"] ?? "not found";
+        static string UseDB = ConfigurationManager.AppSettings["UseDB"] ?? "not found";
+        static IStaffDatabaseHandler DBHandler = new StaffDatabaseSQLHandler(SQLConnectionString);
```

Comment:
Woudn't this fail if the DBConnectionString is set as "not found"? You consider making it an intentional exception